Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.5
```

Comment:
Latest alpine is 3.7 can we use that?

https://alpinelinux.org/posts/Alpine-3.7.0-released.html